.github/workflows/pullrequests.yml

```diff
@@ -102,3 +95,46 @@ jobs:
 
       - name: Report Code Coverage
         uses: codecov/codecov-action@v3
+
+  quarantine:
+    name: "Quarantined Tests"
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v2
```

Comment:
I wonder if you can re-use the common install action we use in `c3` (defined [here](https://github.com/cloudflare/workers-sdk/blob/main/.github/actions/install-dependencies/action.yml)).

```yaml
    steps:
      - name: Checkout Repo
        uses: actions/checkout@v3
        with:
          fetch-depth: 0

      - name: Install Dependencies
        uses: ./.github/actions/install-dependencies

      - name: "Create .env file"
        run: |
          touch .env
          echo TURBO_API=${{ secrets.TURBO_API }} >> .env
          echo TURBO_TEAM=${{ secrets.TURBO_TEAM }} >> .env
          echo TURBO_TOKEN=${{ secrets.TURBO_TOKEN }} >> .env
          echo TURBO_REMOTE_CACHE_SIGNATURE_KEY=${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }} >> .env

   ...
```